lib/Core/Executor.cpp

```diff
@@ -2935,7 +2912,9 @@ void Executor::doDumpStates() {
 }
 
 void Executor::run(ExecutionState &initialState) {
-  bindModuleConstants();
+  if (tries==0)
+		bindModuleConstants(); // Only do this once, because we can reuse it.
```

Comment:
can we move this function outside of the Executor, maybe in klee/main.cpp?

Reply:
I see your point. It requires some refactoring but I'll try :)